app/models/az321_contribution.rb

```diff
@@ -33,6 +33,5 @@ class Az321Contribution < ApplicationRecord
   validates :date_of_contribution,
     inclusion: {
       in: TAX_YEAR.beginning_of_year..TAX_YEAR.end_of_year
-    },
-    presence: true
+    }
```

Comment:
why is the presence check removed?

Reply:
for this the inclusion validation serves as a presence check as well, as `nil` values fall outside of the range. It made it so I didn't have to duplicate the error message